README.md

```diff
@@ -239,6 +241,97 @@ app.post("/messages", async (req, res) => {
 app.listen(3001);
 ```
 
+### HTTP on Edge
+
+For Edge based environments (Vercel, Cloudflare Workers) you can use the SSEEdgeTransport
+which returns a SSE response and accepts normal HTTP requests.
+
+For example using Hono inside of a Cloudflare Worker:
+
+```typescript
+import { Hono } from 'hono';
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { SSEEdgeTransport } from "@modelcontextprotocol/sdk/server/sseEdge.js";
+
+const server = new McpServer({
+  name: "example-server",
+  version: "1.0.0"
+});
+
+// ... set up server resources, tools, and prompts ...
+
+const app = new Hono();
+app.get('/sse', async (c) => {
+  const uuid = crypto.randomUUID();
+  const transport = new SSEEdgeTransport('/messages', uuid);
+  return transport.sseResponse
+});
+
+app.post('/messages', async (c) => {
+  // Note: to support multiple simultaneous connections, these messages will
+  // need to be routed to a specific matching transport. (This logic isn't
+  // implemented here, for simplicity.)
+  await transport.handlePostMessage(req, res);
```

Comment:
The doc is not correct here, maybe we need to pass c(Context) to handlePostMessage?